src/main/resources/application.properties

```diff
@@ -25,9 +25,9 @@ spring.flyway.baseline-version=0
 
 springdoc.api-docs.path=/swagger-ui
 
-minio.url=
-minio.accessKey=
-minio.secretKey=
+minio.url=http://thedb.duckdns.org:9000
```

Comment:
Please don't add secrets in the commits. In intelliJ just before the commit you can untick each row and commit just the ones that you have to.